core/audits/largest-contentful-paint-element.js

```diff
@@ -152,6 +158,9 @@ class LargestContentfulPaintElement extends Audit {
       score: 1,
       displayValue,
       details,
+      metricSavings: {
+        LCP: metricLcp || 0,
```

Comment:
this doesn't seem very meaningful

Reply:
This is why we need to balance the impact ranking with the guidance ranking when we sort audits. This audit will have a consistently high impact but we also gave it a very low guidance ranking: https://github.com/GoogleChrome/lighthouse/pull/15025/files#diff-de507a9267e42ea608e9eab1c02d92e8283d073b43edf8b30fdd9fb2084115ea. The audit doesn't give any guidance, but it is still an extremely useful tool to improve LCP and that should be reflected in the savings. It's a similar situation to https://github.com/GoogleChrome/lighthouse/pull/15070

Currently this audit sits near the bottom of every report. In my personal opinion, elevating this audit in the sort order would be a good thing.

Reply:
I understand how this is a knob to turn to make this audit higher in the future sorting, it just doesn't mesh that this property is called `savings` and conceptually this audit isn't something that would result in savings, or even anything to fix. That's why I suggested at least a comment. It's also a hint that maybe the property `metricSavings` is not the best name.

Reply:
Gotcha, I'll add a comment like you said. Also, `metricImpact` is an alternative that might make more sense conceptually.

> conceptually this audit isn't something that would result in savings, or even anything to fix.

FWIW my thinking is that the LCP element *is* the thing to fix/improve for this audit.

Reply:
It's the "thing" but the savings given is not realistic, because a LCP of zero is not realistic.

Reply:
This is true. WDYT about using `lcp - fcp` as the savings?